src/content/lodash/lang/isNil/lodash.js

```diff
@@ -0,0 +1,11 @@
+// https://lodash.com/docs/#isNil
+import { isNil } from 'lodash'
+
```

Comment:
- `isNil()` checks if the value passed in is `undefined` or `null`, while `isNull()` and `isUndefined` only check for either `undefined` or `null` but not both.